### PR TITLE
Issue#19 car entity

### PR DESCRIPTION
**Pull request by HrBjarup**
Created entity and DTO class for cars

### CODE REVIEW

File: src/main/java/entities/Car.java [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+import java.util.Objects;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.NamedQuery;
+
+
+@Entity
+@NamedQuery(name = "RenameMe.deleteAllRows", query = "DELETE from RenameMe")
```
Comment (Runi-VN) [inline src/main/java/entities/Car.java]: renameMe

Comment (HrBjarup) [reply src/main/java/entities/Car.java]: Oh geez, everytime.. I always forget that

File: src/main/java/entities/Car.java [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+import java.util.Objects;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.NamedQuery;
+
+
+@Entity
+@NamedQuery(name = "CAR.deleteAllRows", query = "DELETE from CAR")
```
Comment (Runi-VN) [inline src/main/java/entities/Car.java]: not sure if uppercase works, but approved :D

Comment (HrBjarup) [reply src/main/java/entities/Car.java]: I'll change it

Comment (HrBjarup) [reply src/main/java/entities/Car.java]: The build is failing anyway

Comment (HrBjarup) [reply src/main/java/entities/Car.java]: Should be ok now

**Pull request merged**